api_tests/base/test_serializers.py

```diff
@@ -76,7 +76,7 @@ def test_registration_serializer(self):
         # fields that are visible for withdrawals
         visible_on_withdrawals = ['contributors', 'date_created', 'description', 'id', 'links', 'registration', 'title', 'type', 'current_user_can_comment']
         # fields that do not appear on registrations
-        non_registration_fields = ['registrations', 'draft_registrations']
+        non_registration_fields = ['registrations', 'draft_registrations', 'templated_from']
```

Comment:
I think if the model keeps the "template_node" on the registration, then it makes sense to display this on the registration, but it shouldn't be visible on withdrawals. So I think drop all the changes on this file and ensure it still works. You may have to change another test.